src/test/java/uk/gov/companieshouse/confirmationstatementapi/interceptor/validation/SubmissionIdValidationInterceptorTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.companieshouse.confirmationstatementapi.interceptor.validation;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.web.servlet.HandlerMapping;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.when;
+import static uk.gov.companieshouse.confirmationstatementapi.utils.Constants.CONFIRMATION_STATEMENT_ID_KEY;
+
+@ExtendWith(MockitoExtension.class)
+class SubmissionIdValidationInterceptorTest {
+
+    @Mock
+    private HttpServletRequest mockHttpServletRequest;
+    @InjectMocks
+    private SubmissionIdValidationInterceptor submissionIdValidationInterceptor;
+
```

Comment:
I'd add a blank string test here as well once the blank check has been added to the interceptor